src/stream_wrap.cc

```diff
@@ -86,7 +86,7 @@ int StreamWrap::GetFD() {
   int fd = -1;
 #if !defined(_WIN32)
   if (stream() != nullptr)
-    fd = stream()->io_watcher.fd;
+    uv_fileno(reinterpret_cast<uv_handle_t*>(stream()), &fd);
 #endif
```

Comment:
On Windows that's a `HANDLE*`, and while it can be cast to an `int` because only 32 bits are used AFAIS, I wasn't sure if we wanted to expose that.